config/config.go

```diff
@@ -26,12 +32,18 @@ func Get() (*Config, error) {
 	}
 
 	cfg := &Config{
-		BindAddr:                   ":25000",
-		RendererURL:                "http://localhost:20010",
-		SearchAPIURL:               "http://localhost:23900",
-		GracefulShutdownTimeout:    5 * time.Second,
-		HealthCheckInterval:        30 * time.Second,
-		HealthCheckCriticalTimeout: 90 * time.Second,
+		BindAddr:                    ":25000",
```

Comment:
Suggest you make this localhost so it doesn't cause the mac firewall Allow/Deny popup on startup. This would then need changes to the unit tests as appropriate.
```suggestion
		BindAddr:                    "localhost:25000",
```